stories/button.stories.js

```diff
@@ -28,18 +29,20 @@ export const Playground = () => ({
       @type={{type}}
       disabled={{disabled}}
       onClick={{onClick}}
+      class={{class}}
     >
-      {{children}}
+      {{content}}
     </DenaliButton>
   `,
   context: {
-    children: text('children', 'Button'),
-    isActive: boolean('isActive', false),
-    disabled: boolean('disabled', false),
-    isInverse: boolean('isInverse', false),
-    onClick: action('onClick'),
-    size: select('size', SIZES),
-    style: select('style', STYLES),
-    type: select('type', TYPES),
+    isActive: boolean('isActive', false, argument),
+    isInverse: boolean('isInverse', false, argument),
+    onClick: action('onClick', argument),
```

Comment:
```suggestion
    onClick: action('onClick', attribute),
```
?

Reply:
Not sure how this is supposed to work but it doesn't seem to be doing anything

Reply:
yep, good catch

Reply:
Actually that is an action instead of knob, let me update that